### ISSUE TITLE
In version 0.2.0 syntax highlighting does not work

### ISSUE THREAD

**Issue by anildigital** [schacon/showoff#31]
For showoff version 0.2.0. Looks like syntax syntax highlighting for Ruby code is not working. I have to revert back to earlier version.

**Comment by davetron5000** [comment]
It is working for me; If I omit the `@@@ Ruby` from before the code, though, it doesn't highlight.  so you have to do

```
!SLIDE

    @@@ Ruby
    puts "foobar" if quux
    puts "quux"
```

**Comment by anildigital** [comment]
Looks like it got fixed in version 0.2.1. But definitely broken with version 0.2.0

**Comment by anildigital** [comment]
closing it
